src/Components/FormElement.js

```diff
@@ -1,38 +1,65 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Input from './Input'
 export default function FormElement(props) {
+
+  const handleSubmit = async (event)=>{
+    event.preventDefault();
+    const headers = new Headers({
+      'Content-Type': values["Content-Type"],
+      'X-HS-AccessKey': values["X-HS-AccessKey"]
+    })
+    try{
+      await fetch(`http://localhost:8080/v2/partner/subscription/${values["SubscriptionId"]}`, {
+      method: 'GET',
+      headers,
+      mode: 'cors'
+         })
+         .then(results => console.log(results))}
+
+    catch(error){console.error(error)}
+
+  }
+  const [values, setValues] = useState({})
   return (
     <form>
         <h2>{props.title}</h2>
         <br/>
         <h3>{props.description}</h3>
         <br/>
         <br/>
-        {props.obj["pathParams"] && props.obj["pathParams"].map((element, idx) => (
+        {props.obj["pathParams"] && props.obj["pathParams"].map((element) => (
           <>
-          <Input name = {element.name} inputType = {element.inputType} placeholder = {element.name} type={element.type} requestContentType={element.requestContentType}/>
+          <Input values = {values} setValues = {setValues} name = {element.name} inputType = {element.type} placeholder = {element.name} type={element.type} requestContentType={element.requestContentType}/>
```

Comment:
const mapInputValues = (element) => {
 const {name, type, requestContentType} = element
 return {
  name: name,
  placeholder: name
  .......
 }
}


....
const inputProps = mapInputProps(element)
<Input {...inputProps}/>

Reply:
Made the function but not used till now, because of the config changes in the main branch.